catalog/spatial/geocoding/spatial-geocoding-localsearch/src/main/java/org/codice/ddf/spatial/geocoder/geonames/GeoNamesLocalIndex.java

```diff
@@ -60,13 +60,14 @@ public GeoResult getLocation(final String location) {
                         population);
             }
         } catch (GeoEntryQueryException e) {
-            LOGGER.error("Error querying the local GeoNames index", e);
+            LOGGER.warn(
```

Comment:
According to the ticket, do we still want to the stack trace printing out, or would something like e.getMessage() be acceptable?